### ISSUE TITLE
Support internationalization

### ISSUE THREAD

**Issue by futurely** [NVIDIA/DIGITS#167]
The objective is to display the GUI in languages other than Enligsh. [Flask-Babel](https://pythonhosted.org/Flask-Babel/) is a potential solution.

**Comment by lukeyeager** [comment]
Good idea and thanks for the specific package suggestion. This *is* something we should get around to eventually.

Do you have a specific language you'd like to see, or are you just making a general suggestion?

**Comment by futurely** [comment]
Chinese. There're thousands of users of both Caffe and NVIDIA GPU in China.

**Comment by lukeyeager** [comment]
好，感谢您的建议。
